0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -164,6 +166,13 @@ The sender uses the receiver details to create the ILP packet:
 
 Note that the sender can send as many payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
 
+### Payment Result
+
+When the receiver fulfills a payment that was set up through spsp, they may add a signature of the OER-encoded fulfillment data packet using the `signature_info` provided.
```

Comment:
Can you clarify this one? How would anyone forge a fulfillment?

Reply:
I don't think that faking successful payments to make your success rate look better is an actual problem. Sending payments is what ILP is made for, so if a connector wants to send payments to himself so be it.

> Now you create fake traffic that goes to your customers, they forward it via you, and you fulfill all this fake traffic. That makes it look like your success rate is higher than it really is.

I don't agree that this is "fake traffic". Since the payments are fulfillment, these are really just ordinary payments.